Makefile.codegen

```diff
@@ -57,17 +57,6 @@ codegen-openapi: build-codegen
 	@echo "Generating OpenAPI structs for Kubernetes Clients"
 	./build/$(CODE_GEN_BIN_NAME) --generator-version $(OPENAPI_GEN_VERSION) openapi --output-package=pkg/client --input-package=github.com/jenkins-x/jx/pkg/apis --group-with-version=jenkins.io:v1
 
-generate-clean: clean ## Delete code generation tools
```

Comment:
hmm, not so sure this is a good idea. This ensured you never had "stale" tools.

Reply:
ok - let me do that properly.

Reply:
https://github.com/jenkins-x/jx/pull/3977/files#diff-e1f8204ecc7085af67da9ba2d35cc266R85